homeassistant/components/shelly/config_flow.py

```diff
@@ -363,8 +363,10 @@ def async_get_options_flow(config_entry: ConfigEntry) -> OptionsFlowHandler:
     @callback
     def async_supports_options_flow(cls, config_entry: ConfigEntry) -> bool:
         """Return options flow support for this handler."""
-        return config_entry.data.get("gen") == 2 and not config_entry.data.get(
-            CONF_SLEEP_PERIOD
+        return (
+            config_entry.data.get("gen") == 2
+            and not config_entry.data.get(CONF_SLEEP_PERIOD)
+            and config_entry.data.get("model") != "SAWD-0A1XX10EU1"
```

Comment:
I think it would be better to move the model number to a const. In https://github.com/home-assistant/core/pull/103937 you also refer this model number and if Shelly create a new variant it would be harder to track and fix all the places.

Reply:
Yeah I realized this morning that all three opened PRs are more or less related to WallDisplay and we need a constant with model number. I’ll update PRs today.